### PR TITLE
Added postcode search AT

### PR DESCRIPTION
**Pull request by ryangrundy7**
# Motivation and Context

The first feature needed for ops-ui is to be able to search for cases via a postcode. 
# What has changed

- Added new AT for postcode search on rops ui

# How to test?

- Run ATs with other branches.
- Run tests in GCP

# Links

[Trello](https://trello.com/c/qkkjvupd)

### CODE REVIEW

**Comment by ryangrundy7** [comment]
[R Ops - search for case and show results (13)](https://trello.com/c/qkkjvupd/1221-r-ops-search-for-case-and-show-results-13)

**Review by LeoKHoward — APPROVED** [review]
Works fine for me

**Review by NickGrantX — APPROVED** [review]

**Review by liamtoozer — CHANGES_REQUESTED** [review]
acceptance_tests_pod.yml env vars need to match the ROPS ones

File: acceptance_tests_pod.yml [hunk 1]
```diff
@@ -149,6 +149,10 @@ spec:
       value: "case-api"
     - name: CASEAPI_SERVICE_PORT
       value: "80"
+    - name: CASEAPI_SERVICE_HOST
```
Comment (liamtoozer) [inline acceptance_tests_pod.yml]: Just need to change these

Comment (ryangrundy7) [reply acceptance_tests_pod.yml]: Done

**Review by liamtoozer — APPROVED** [review]

**Pull request merged**